changelog

```diff
@@ -1,3 +1,244 @@
+        * New missions offered by the Quarg. (@Arachi-Lover)
+        * Created two short spaceport missions in human space. (@Nameson2)
+        * Created a difficult combat mission only available to early pilots. (@Anarchist2)
+        * The player will now be warned when their combat rating is high enough that Marauder Hunted fleets will start showing up. (@Amazinite)
+        * Created jobs and missions available on Kor Efreti worlds. (@W1zrad)
+        * Created new bounty hunting jobs to track down disguised ships. (@Anarchist2)
+        * Added a new string of missions during the start of the Wanderer campaign that leads to a sideplot to help the Hai build a new ship. (@Amazinite, @Arachi-Lover, @beccabunny, @LocalGod79, @Zitchas)
+        * Created an epilogue mission for Barmy Edward. (@beccabunny, @Darcman99)
+        * Created a mission where you can inform the Remnant about the research that the Deep are doing on the Ember Waste. (@Amazinite, @Zitchas)
+      * New outfits and ships:
+        * Created an ammo storage outfit for Finisher torpedoes. (@Arachi-Lover)
+        * Ka'het ships now have power generation and storage outfits. (@beccabunny)
+        * The Navy now has a unique logistics vessel, the Auxiliary, along with a new fighter, the Dropship. (@Brick63)
+        * Added a new Remnant ship, the Peregrine. (@beccabunny, @Zitchas)
+        * Created an ammo storage outfit for EMP torpedoes. (@Anarchist2)
+        * Ka'het ships now have engine outfits. (@beccabunny, @Karirawri, @TheronEpic)
+        * Created six new Heliarch ships. (@Arachi-Lover)
+        * Added a new Ka'het ship with a unique superweapon and other new outfits. (@beccabunny, @Karirawri)
+      * Other new content:
+        * The Penguin now uses the new engine flare mechanics. (@Zitchas)
+        * Added new civilian hails. (@Anarchist2, @Naminson, @petervdmeer)
+        * New logs for various characters, factions, and plot points. (@MasterOfGrey, @pega3)
+        * Added a new Builder station to the Graveyard. (@beccabunny)
+        * Wrote more news messages for human space and beyond. (@Amazinite, @Anarchist2, @Arachi-Lover, @Galaucus, @petervdmeer, @Pointedstick, Swiren#8879, @W1zrad, @WarlordMike, @Zitchas)
+        * Wrote friendly and hostile hails for the Remnant. (@temtemy)
+        * The Archon now has a ship description. (@Zitchas)
+        * Added three new person ships. (@beccabunny, @Brick63, @MCOfficer, @Zitchas)
+        * Quarg outfits now have placeholder descriptions. (@Amazinite)
+        * Planets in the Graveyard region now gain expanded descriptions after you learn more about the region. (@beccabunny)
+        * Added system attributes to various system for use in missions. (@Galaucus)
+        * Expanded the list of potential Heliarch ship names. (@Arachi-Lover)
+        * Scouts will now actually spawn in merchant fleets. (@ScruffyKitty)
+        * Added various new help messages describing hidden features. (@Terin)
```

Comment:
```suggestion
        * Added various new help messages describing hidden features. (@Terin)
        * Added gravitational and hazard effects to Sagittarius A* so that it is now dangerous. (@Zitchas)
```

Reply:
If we're adding Sag A* to the content list, why not the rest of the EW hazards?

Reply:
Ember Waste hazards are mentioned under big changes.

Reply:
While the Ember Wastes are (appropriately) listed under big hazards, I felt that Sag A* hazards might warrant a separate listing under minor changes. Most people probably don't associate the two together, so are unlikely to realize that a change for one means the other changed.)